scintillometry/squaring/base.py

```diff
@@ -0,0 +1,25 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import numpy as np
+
+from ..base.base import TaskBase
+
+
+__all__ = ['SquareTask']
+
+
+class SquareTask(TaskBase):
+
+    def __init__(self, ih, samples_per_frame=1):
```

Comment:
`samples_per_frame` does not seem to be used.

Reply:
True - removed.  Not 100% sure we should even be passing `ih.samples_per_frame` on, but at least that makes more sense than `1`.